README.md

```diff
@@ -179,6 +179,17 @@ fastify.register(require('fastify-basic-auth'), {
 })
 ```
 
+### `header` String (optional)
+
+When supplied, the header option is the name of the header to get
+credentials from for validation.
+
+```js
+fastify.register(require('fastify-basic-auth'), {
+  validate,
+  header: 'x-forwarded-authorization'
```

Comment:
could we mention that it must be lowercase or add a lowercase function?

Reply:
why it must be lowercased?

Reply:
Otherwise the lookup of the header will fail.

Reply:
I've added the `.toLowerCase()` suggestion from @mcollina. Will update the test to reflect this.

Reply:
Updated the test to demonstrate the to lowercasing. Not sure it needs a whole test of it's own but let me know if that's not enough.